docs/clusters/eiger.md

```diff
@@ -44,18 +42,16 @@ Eiger is an Alps cluster that provides compute nodes and file systems designed t
 
 ### Compute nodes
 
-!!! under-construction
-    During this Early Access phase, there are 19 compute nodes for you to test and port your workflows to the new Eiger deployment. There is one compute node in the `debug` partition and one in the `xfer` partition for internal data transfer.  The remaining compute nodes will be moved from `eiger.cscs.ch` to `eiger.alps.cscs.ch` at a later date (provisionally, 1 July 2025).
-
-Eiger consists of 19 [AMD Epyc Rome][ref-alps-zen2-node] compute nodes. 
-
-There is one login node, `eiger-ln010`. 
-
-[//]: # (TODO: You will be assigned to one of the four login nodes when you ssh onto the system, from where you can edit files, compile applications and start simulation jobs.)
-
-| node type | number of nodes | total CPU sockets | total GPUs |
-|-----------|-----------------| ----------------- | ---------- |
-| [zen2][ref-alps-zen2-node] | 19 | 38      | -          |
+Eiger consists of multicore [AMD Epyc Rome][ref-alps-zen2-node] compute nodes: please note that the total number of available compute nodes on the system might vary over time, therefore you might want to check them with the Slurm command `sinfo -s`.
+```
```

Comment:
Instead of adding this on the Eiger page, how about adding a section to the Slurm documentation about how to inspect the number of available nodes on a cluster.

That way we can link to it from all of the vCluster pages?

The slurm docs:
https://docs.cscs.ch/running/slurm/

The source for them is in `/docs/running/slurm.md`